content/slack-invite.html

```diff
@@ -0,0 +1,10 @@
+title=Slack invite redirect
+date=2016-07-26
+type=page
```

Comment:
I think using `page` it adds in the resulting page the general HTML structure with menu, footer, etc. it does not?

Reply:
Yes, it does. Ideally, we'd have to create a new light template for that, I think. If type is set to html, as it was in the first commit, jbake finds slack-invite.html, but does not create it in the output folder, so it throws Reason: Not Found if you try to access the URL.

Reply:
Well spotted